test/run_tests.sh

```diff
@@ -142,8 +156,8 @@ function run_op_tests {
   run_pjrt python3 "$CDIR/test_operations.py" "$@" --verbosity=$VERBOSITY
   run_test python3 "$CDIR/test_async_closures.py"
   run_test python3 "$CDIR/test_xla_dist.py"
-  run_test python3 "$CDIR/test_profiler.py"
-  run_test python3 "$CDIR/test_ops.py"
+  # run_test python3 "$CDIR/test_profiler.py"
+  # run_test python3 "$CDIR/test_ops.py"
```

Comment:
hmm I felt like we should not just disable these tests?

Reply:
disable test_profiler.py and test_ops .py according to the log:
https://pantheon.corp.google.com/cloud-build/builds/9b7f9f73-a936-4d1d-b071-fe419c232c90?project=tpu-pytorch&jsmode=o&mods=allow_workbench_image_override

Reply:
my point is if you do this, you disable this test on both CPU and GPU CI. You can skip them in TPU CI for now and debug them later.

Reply:
Thanks, Jack, got it, and updated